stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -80,4 +328,61 @@ private static String capitalize(String str) {
     buffer.setCharAt(0, Character.toTitleCase(buffer.charAt(0)));
     return buffer.toString();
   }
+
+  private final class FieldBackedCSSProperty extends ViewCSSProperty {
+    private final Field mField;
+
+    public FieldBackedCSSProperty(
+        Field field,
+        String cssName,
+        @Nullable ViewDebug.ExportedProperty annotation) {
+      super(cssName, annotation);
+      mField = field;
+      mField.setAccessible(true);
+    }
+
+    @Override
+    public Object getValue(View view) throws InvocationTargetException, IllegalAccessException {
+      return mField.get(view);
+    }
+  }
+
+  private final class MethodBackedCSSProperty extends ViewCSSProperty {
+    private final Method mMethod;
+
+    public MethodBackedCSSProperty(
+        Method method,
+        String cssName,
+        @Nullable ViewDebug.ExportedProperty annotation) {
+      super(cssName, annotation);
+      mMethod = method;
+      mMethod.setAccessible(true);
+    }
+
+    @Override
+    public Object getValue(View view) throws InvocationTargetException, IllegalAccessException {
+      return mMethod.invoke(view);
+    }
+  }
+
+  private abstract class ViewCSSProperty {
+    private final String mCSSName;
+    private final ViewDebug.ExportedProperty mAnnotation;
+
+    public ViewCSSProperty(String cssName, @Nullable ViewDebug.ExportedProperty annotation) {
+      mCSSName = cssName;
+      mAnnotation = annotation;
+    }
+
+    public final String getCSSName() {
+      return mCSSName;
+    }
+
+    public abstract Object getValue(View view)
+        throws InvocationTargetException, IllegalAccessException;
```

Comment:
since we always call `setAccessible(true)`, do we still need `IllegalAccessException` in the signature ?